frontend/packages/dev-console/integration-tests/support/commands/hooks.ts

```diff
@@ -19,7 +20,14 @@ before(() => {
   const bridgePasswordPassword: string = Cypress.env('BRIDGE_HTPASSWD_PASSWORD') || 'test';
   cy.login(bridgePasswordIDP, bridgePasswordUsername, bridgePasswordPassword);
   cy.document().its('readyState').should('eq', 'complete');
+  cy.exec(
+    `oc patch console.operator.openshift.io/cluster --type='merge' -p '{"spec":{"customization":{"perspectives":[{"id":"dev","visibility":{"state":"Enabled"}}]}}}'`,
+    { failOnNonZeroExit: false },
```

Comment:
Why wouldn't be fail on non-zero codes?

Reply:
This was used for the case that each package(ODC) will be using this patch so to avoid any issue I used this, I'll update it